src/api/java/com/minecolonies/api/inventory/InventoryCitizen.java

```diff
@@ -235,6 +235,10 @@ public boolean hasCustomName()
     @Override
     public ItemStack getStackInSlot(final int index)
     {
+        if (index == NO_SLOT)
```

Comment:
What is NO_SLOT?
The contract specifies that index needs to be between 0 and at most the count of slots.

Reply:
This happens when the heldItemSlot = -1, should I move this in the getHeldItem instead?